### PR TITLE
Added notification wrapper service and required changes

### PR DESCRIPTION
**Pull request by manojkrsaini09**
Added wrapper for notification service.

### CODE REVIEW

File: src/main/resources/public/module/controllers/createCustomerController.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ function($scope,$state,CustomerDataService,LocationDataService,Notification,$tra
 	$scope.saveNewCustomer = function(isAddProduct){
 		if($scope.CustomerForm.$invalid){
 			 $scope.formValidate = true;
-			 Notification.error({title: 'Warning', message: $filter('translate')('messages.customer.form.validation.error'), delay: 2000});
```
Comment (aayushKhandpur) [inline src/main/resources/public/module/controllers/createCustomerController.js]: Need option to set both title message and decsription message sent by server

File: src/main/resources/public/module/services/notification-wrapper-service.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+angular.module('app')
+.factory('NotificationWrapperService',NotificationWrapperService);
+
+NotificationWrapperService.$inject = ['Notification','$translate','$filter'];
+
+function NotificationWrapperService(Notification,$translate,$filter){
+	var NotificationService = {};
```
Comment (aayushKhandpur) [inline src/main/resources/public/module/services/notification-wrapper-service.js]: All constant value fields should be in CAPITAL.

Comment (manojkrsaini09) [reply src/main/resources/public/module/services/notification-wrapper-service.js]: done

File: src/main/resources/public/module/controllers/createCustomerController.js (hunk 1 above)
Comment (manojkrsaini09) [reply src/main/resources/public/module/controllers/createCustomerController.js]: Added parameter for title.

**Comment by manojkrsaini09** [comment]
@aayushKhandpur review changes are done

**Pull request merged**